### ISSUE TITLE
HammerLib Crash w/ Polymorph Mod Added

### ISSUE THREAD

**Issue by JSmurf11** [dragon-forge/HammerLib#10]
So I was making a modpack for me and my friends, everything was going well until I added Polymorph mod which makes HammerLib crash minecraft, it doesn't even reach the loading screen of minecraft (white bar). Is it incompatibility?

Here is the crash report: https://pastebin.com/4GMin7EM

![hammerlib](https://user-images.githubusercontent.com/86183017/163786612-b22fd5c3-a74f-4588-8a3d-c302b3b6341b.png)

**Comment by niftybottle** [comment]
I am also getting this error, on 1.18.1

**Comment by niftybottle** [comment]
However, Polymorph seems to have nothing to do with it - I've narrowed it down to the most recent version of Structurize (1.0.387), and a test setup with only Hammer Lib and Structurize (no polymorph) installed reproduces the crash. Downgrading to Structurize 1.0.342 (the version I had before I updated, as well as the version required for Minecolonies) fixed the crash. It seems likely this is on their end, as they are the ones who updated, so I'll submit this crash to them.

Crash Log, for reference: https://pastebin.com/hQUdzgBx
